### PR TITLE
Add a fieldValidator to linkedin url format

### PR DESCRIPTION
**Pull request by shegunmb**
Check if a string is a valid linkedin url

### CODE REVIEW

File: src/__tests__/fieldValidators.js [hunk 1]
```diff
@@ -156,6 +157,28 @@ it('testEmailFormat', () => {
...
+    let actual = testLinkedinUrlFormat('https://www.linkedin.com/in/url-test/');
+    let expected = null;
+    expect(actual).toBe(expected);
+
+    actual = testLinkedinUrlFormat('Bob');
+    expected = 'notLinkedinUrl';
+    expect(actual).toBe(expected);
+
+    actual = testLinkedinUrlFormat(null);
+    expected = null;
```
Comment (T0din) [inline src/__tests__/fieldValidators.js:170]: Why would you expect null if there's no value ?

Comment (shegunmb) [reply src/__tests__/fieldValidators.js:170]: It doesn't check if the field is required, it's the job of `validateRequired`

Comment (T0din) [reply src/__tests__/fieldValidators.js:170]: Sounds good to me ;)

File: src/fieldValidators.js [hunk 2]
```diff
@@ -78,6 +78,18 @@ const testEmailFormat = (value: ?string) => {
     return regexp.test(value.trim()) && !regexpAccent.test(value.trim()) ? null : 'notEmail';
 };
 
+/**
+ * @desc Test if a value is a valid linkedin url
+ * @param {?string} value - The value of the field
+ * @return {?string} 'notLinkedinUrl' if the value is not a valid linkedin Url, null if it is
+ */
+const testLinkedinUrlFormat = (value: ?string) => {
```
Comment (alefevre19) [inline src/fieldValidators.js]: On n'écrit pas du `LinkedIn` plutôt que `Linkedin` d'habitude ?

Comment (shegunmb) [reply src/fieldValidators.js]: Yes I fix that

**Pull request merged**